src/swagger-specs/query-service.yaml

```diff
@@ -330,10 +333,10 @@ paths:
           description: "Internal Server Error"
 
     patch:
-      operationId: "updateSchedule"
+      operationId: "patchSchedule"
       tags:
         - "Schedules"
-      summary: Update a schedule
+      summary: Partially update a schedule
```

Comment:
The distinction between patch and update (and specifying that the former is just a partial update) is only relevant if your API has _both_ PUT and PATCH methods for the same endpoint. If it only uses one or the other (as is the case with QS), the verb should always be `update`, regardless of whether it is a PUT or PATCH request. Same applies to the summary--this should remain as "Update a schedule"